### ISSUE TITLE
json_report ?

### ISSUE THREAD

**Issue by jvermillard** [CrowdStrike/container-image-scan-action#16]
I would like to export the json report, I tried the json_report from the doc but:

```
Warning: Unexpected input(s) 'json_report', valid inputs are ['falcon_client_id', 'container_repository', 'container_tag', 'crowdstrike_region', 'crowdstrike_score', 'retry_count']
Run crowdstrike/container-image-scan-action@v0.5
  with:
    falcon_client_id: xxxxx
    crowdstrike_region: us-2
    container_repository: 1111111[55](https://github.com/xxx/xxx/runs/11111111?check_suite_focus=true#step:4:55)x.dkr.ecr.eu-west-1.amazonaws.com/xxx
    container_tag: latest
    crowdstrike_score: 2000
    json_report: /tmp/report.json
    retry_count: 10
  env:
    FALCON_CLIENT_SECRET: ***
```

Is it a real option?

**Comment by isimluk** [comment]
Hello @jvermillard,

Thanks for your report.

I think the issue was the `json_report` was added recently and it was not present in the released `@v0.5` version.

I just relased 0.6. Would you be willing to retry with `@v0.6` suffix added to your gh action spec?

**Comment by jvermillard** [comment]
It works with the new release. Thanks
